src/main/kotlin/no/nav/familie/ba/sak/ekstern/skatteetaten/SkatteetatenService.kt

```diff
@@ -53,23 +53,23 @@ class SkatteetatenService(
         LOG.info("Fant ${perioderFraInfotrygd.size} skatteetatenperioder fra infotrygd")
 
         val samletPerioder = mutableListOf<SkatteetatenPerioder>()
-        unikePersoner.forEach {
+        unikePersoner.forEach {person ->
             val resultatInfotrygdForPerson =
-                perioderFraInfotrygd.firstOrNull { perioder -> perioder.ident == it }
-            val perioderFraInfotrygd =
+                perioderFraInfotrygd.firstOrNull { perioder -> perioder.ident == person }
+            val perioderFraInfotrygdForPerson =
                 resultatInfotrygdForPerson?.perioder ?: emptyList()
 
-            val resutatBaSakForPerson = perioderFraBaSak.firstOrNull { perioder -> perioder.ident == it }
+            val resutatBaSakForPerson = perioderFraBaSak.firstOrNull { perioder -> perioder.ident == person }
```

Comment:
Når du først er på feilretting her 😆
```suggestion
            val resultatBaSakForPerson = perioderFraBaSak.firstOrNull { perioder -> perioder.ident == person }
```

Reply:
Burde kanskje gjøre det i intelliJ så stedene som refererer til denne også blir oppdatert